boards/arm/twr_ke18f/twr_ke18f.dts

```diff
@@ -146,6 +146,11 @@
 	sample-time = <12>;
 };
 
+&can0 {
+	status = "ok";
+	bus-speed = <1000000>;
```

Comment:
We use a standard speed fo 125000. This should be changed to be compatible with other board when running the sample code.

Reply:
Fixed.